tests/redirects/test_urls.py

```diff
@@ -46,6 +46,7 @@ def test_x_robots_tag(base_url):
     '/firefox/channel/',
     '/firefox/desktop/',
     '/firefox/developer/',
+    '/firefox/enterprise/',
```

Comment:
Are we sure this is necessary? Do we have redirects to test against this URL?

Reply:
I'm not sure it's necessary. I was mostly looking at what had been done to set up /developer/ and copying that.

I thought it might test this line in urls.py:
```
page('firefox/enterprise', 'firefox/enterprise/index.html'),
```

Reply:
I'm *pretty* sure we only need to add URLs here that have redirects applied elsewhere in bedrock.

@pmac Can you confirm?

Reply:
Removing for now.